io/src/hdl_grabber.cpp

```diff
@@ -472,8 +472,8 @@ pcl::HDLGrabber::enqueueHDLPacket (const std::uint8_t *data,
 {
   if (bytesReceived == 1206)
   {
-    std::uint8_t *dup = static_cast<std::uint8_t *> (malloc (bytesReceived * sizeof(std::uint8_t)));
```

Comment:
Please revert to `malloc` -- this is later freed with `free()` and we don't want a `new`-`free`-mismatch